### PR TITLE
DT-169 Nearby routes

### PR DESCRIPTION
**Pull request by hannesj**
[DT-169 View nearby routes](https://digitransit.atlassian.net/browse/DT-169)

### CODE REVIEW

**Comment by Macroz** [comment]
Mukana nyt myös [DT-170 Filter duplicate routes from view](https://digitransit.atlassian.net/browse/DT-170)

File: app/component/route/route-list-container.cjsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+module.exports = Relay.createContainer(RouteListContainer,
+  fragments: queries.RouteListContainerFragments
+  initialVariables:
+    lat: null
+    lon: null
+    radius: 2000
+    numberOfStops: STOP_COUNT
+    agency: config.preferredAgency
+    bucketSize: 100
```
Comment (hannesj) [inline app/component/route/route-list-container.cjsx]: This probably shouldn't be a relay variable?

Comment (Macroz) [inline app/component/route/route-list-container.cjsx]: We should probably have a discussion about which parts are configurable and how.

**Comment by Macroz** [comment]
And now implementation of [DT-171 Filter routes by mode](https://digitransit.atlassian.net/browse/DT-171)

**Pull request merged**